packages/terra-consumer-layout/package.json

```diff
@@ -38,7 +38,8 @@
     "terra-i18n": "^1.5.0",
     "terra-icon": "^1.5.0",
     "terra-overlay": "^1.7.0",
-    "terra-responsive-element": "^1.7.0"
+    "terra-responsive-element": "^1.7.0",
+    "terra-alert": "^1.9.1"
```

Comment:
this can be a dev dependency now that its only used in a test. You can disable the lint rule in that particular file if needed